subsys/zbus/zbus.c

```diff
@@ -191,21 +201,175 @@ static inline int _zbus_vded_exec(const struct zbus_channel *chan, k_timepoint_t
 	return last_error;
 }
 
+#if defined(CONFIG_ZBUS_PRIORITY_BOOST)
+
+static inline void _zbus_chan_update_hop(const struct zbus_channel *chan)
```

Comment:
don't use `_` prefix functions please, it's reserved for the library or something IIRC